course_content/cartopy_course/cartopy_intro.ipynb

```diff
@@ -119,29 +143,157 @@
+    "To do this, we use the `transform` keyword in the plt.plot function :\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = plt.axes(projection=proj)\n",
+    "ax.coastlines()\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the plotted line is bent :  It is a straight line ***in the coordinate system it is defined in***, so that makes it a curved line on this map.\n",
+    "\n",
+    "Also note that, unless we specify a map extent, the map zooms to contain just the plotted data.  \n",
+    "A very simple alternative to that is to plot the 'full map', by calling the **``set_global``** method on the Axes, as in this case :"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ax = plt.axes(projection=proj)\n",
+    "ax.coastlines()\n",
     "ax.set_global()\n",
-    "plt.plot([-100, 50], [25, 25], linewidth=4, transform=ccrs.Geodetic())\n",
+    "plt.plot([x0, x1], [y0, y1], linewidth=4, transform=ccrs.PlateCarree())\n",
     "plt.show()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Notice that unless we specify a map extent (we did so via the **``set_global``** method in this case) the map will zoom into the range of the plotted data."
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Suggested Activity :</font></b> Try modifying the above code to plot a different projection.\n",
+    "\n",
+    "Try re-plotting the \"failed plot\" above, where we did not specify the transform.<br>\n",
+    "But now, add a call to the `set_global` method, to show the full map extent.\n",
+    "\n",
+    "What does this tell you about what was actually being plotted in that case ?\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#\n",
+    "# edit space for user code ...\n",
+    "#"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# SAMPLE SOLUTION\n",
+    "# %load solutions/cartopy_exercise_2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-warning\">\n",
+    "    <b><font color='brown'>Suggested Activity :</font></b> Try taking more control of the plot region,  \n",
+    "<p>using the <a href=\"https://scitools.org.uk/cartopy/docs/latest/matplotlib/geoaxes.html?highlight=set_extent#cartopy.mpl.geoaxes.GeoAxes.set_extent\"> Geoaxes.set_extents</a> method. </p>\n",
+    "( look this up ! )\n",
+    "\n",
+    "What is the _coordinate system_ of the coordinate values which you pass into this method ?\n",
```

Comment:
The formatting of this is a bit strange. It would read as:
>Try taking more control of the plot region,
>using the Geoaxes.set_extents method.
>( look this up ! ) What is the _coordinate system_ of the coordinate values which you pass into this method ? 

Why not:
> Try taking more control of the plot region, using the Geoaxes.set_extents method.
> What is the _coordinate system_ of the coordinate values which you pass into this method ?

Reply:
The "( look this up ! )" is on a new line, and there is a space before the final question.
Does it not do that for you, or do you just not like it ?

Reply:
Sorry I was looking at the formatting on [GitHub renderings of this file](https://github.com/pp-mo/scitools-courses/blob/odd_cartopy_changes/course_content/cartopy_course/cartopy_intro.ipynb), which apparently isn't rendering it correctly. But from the look of the `\n` in the code it looks like it should what you say

I'm happy with the "look this up" being on its own line. It just looked a bit strange in the formatting I was looking at.